src/components/info/feature-info.vue

```diff
@@ -0,0 +1,89 @@
+<script setup lang="ts">
+import { defineProps, onUnmounted } from 'vue'
+import DefaultTemplate from '@/components/info/templates/default-template.vue'
+import LignesBusTemplate from './templates/lignes-bus-template.vue'
+import {
+  FeatureInfoJSON,
+  FeatureJSON,
+} from '../../services/info/feature-info.model'
+import { featureInfoLayerService } from '../../services/info/feature-info-layer.service'
+import {
+  exportFeatureService,
+  ExportFormat,
+  FeatExport,
+} from '@/services/export-feature/export-feature.service'
+import { Feature } from 'ol'
+import { Geometry } from 'ol/geom'
+import GeoJSON from 'ol/format/GeoJSON'
+import useMap, { PROJECTION_LUX } from '@/composables/map/map.composable'
+import { useFeatureInfoStore } from '@/stores/feature-info.store'
+
+defineProps({
+  content: {
+    type: Array as () => FeatureInfoJSON[],
+    required: true,
+  },
+})
+const map = useMap().getOlMap()
+const { clearContent } = useFeatureInfoStore()
+
+onUnmounted(() => {
+  clearContent()
+  featureInfoLayerService.clearFeatures()
+})
+
+const getTemplateComponent = (template: string) => {
+  switch (template) {
+    case 'default.html':
+      return DefaultTemplate
+    case 'lignes_bus.html':
+      return LignesBusTemplate
+    default:
+      return DefaultTemplate
+  }
+}
+
+function onExport(payload: { feature: FeatureJSON; format: ExportFormat }) {
+  if (payload.feature.attributes && !payload.feature.properties) {
+    payload.feature.properties = payload.feature.attributes
+  }
+  const olFeature = new GeoJSON().readFeature(payload.feature, {
+    dataProjection: PROJECTION_LUX,
+    featureProjection: map.getView().getProjection(),
+  })
+  exportFeatureService.export(
```

Comment:
ah maybe we could improve the exporter (in an another PR) and make it accept both types Features and GeoJson features.

Reply:
Yep, right, was happy it was already easy to use for now :-)